content/docs/extension-author-guide/create-extension/setup-local-environment.md

```diff
@@ -69,7 +72,11 @@ We unset any custom lifecycle image that may have been provided to ensure that t
 
 <!-- test:exec -->
 ```bash
-git clone https://github.com/buildpacks/samples.git
+# FIXME: remove branch when https://github.com/buildpacks/samples/pull/149 is merged
+git clone https://github.com/buildpacks/samples.git && \
+  cd samples && \
```

Comment:
shouldn't we also add git fetch origin runext/update-samples command

Reply:
Added in 79ab9738